tests/run.rs

```diff
@@ -880,16 +880,16 @@ mod unmocked_commands {
             r##"
                 |protocols:
                 |  - protocol:
-                |    - /usr/bin/dirname dir/file
-                |    - /bin/ls dir
+                |    - dirname dir/file
+                |    - ls dir
```

Comment:
Haven't reviewed some of the other changes in this PR fully quite yet, but I might leave these existing tests as is? That way if there is a regression regarding path resolution at some point, it is easier to pinpoint why tests are failing

Reply:
Unfortunately the tests fail on my machine if I do not remove the paths from most of the integration test protocol steps.